packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -589,29 +1133,33 @@ class _RenderCupertinoDialog extends RenderBox {
 
   @override
   List<DiagnosticsNode> debugDescribeChildren() => <DiagnosticsNode>[
-    if (contentSection != null) contentSection!.toDiagnosticsNode(name: 'content'),
-    if (actionsSection != null) actionsSection!.toDiagnosticsNode(name: 'actions'),
-  ];
+        if (contentSection != null) contentSection!.toDiagnosticsNode(name: 'content'),
+        if (actionsSection != null) actionsSection!.toDiagnosticsNode(name: 'actions'),
+      ];
```

Comment:
The formatting should stay as it was originally, the indentation should be 2.  See [here in the style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#indent-multi-line-argument-and-parameter-lists-by-2-characters) (there is an example of a List just below that section).

Does your IDE have a formatter that ran on this code maybe?  I'll try to keep an eye out for other formatting changes as I review this.

Reply:
No, I disabled all formatting, probably this was done by my mistake.